tcms/core/widgets.py

```diff
@@ -40,3 +40,35 @@ class Media:
             "prismjs/plugins/autoloader/prism-autoloader.min.js",
             "js/simplemde_security_override.js",
         ]
+
+
+class DurationWidget(forms.Textarea):
+
+    def render(self, name, value, attrs=None, renderer=None):
+        rendered_duration = super().render(name, value, attrs, renderer)
```

Comment:
Remove the call to `super()`. It looks like this is adding an unwanted `textarea` HTML tag above the input+script combination.

Reply:
Does it work when the hidded textarea is not there ?